src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderResourceIntTest_getOffenderAlerts.java

```diff
@@ -53,29 +53,30 @@ void willReturnAllActiveAlertsWhenNoFilter() {
             final var jsonContent = getBodyAsJsonContent(response);
             assertThat(jsonContent).extractingJsonPathArrayValue("$").hasSize(3);
 
-            assertThat(jsonContent).extractingJsonPathBooleanValue("[0].active").isEqualTo(true);
```

Comment:
all a consequence of the alert test data changes to demonstrate the default ordering